spec/controllers/sources_controller_spec.rb

```diff
@@ -16,38 +19,39 @@ def valid_attributes
     attributes_for(:source, partner_id: partner.id)
   end
 
-  before(:each) do
-    sign_in user
-  end
-
   describe 'GET index' do
     it 'assigns all sources as @sources' do
-      source = Source.create! valid_attributes
       get :index
+
       expect(assigns(:sources)).to eq([source])
     end
   end
 
+  describe 'GET show' do
```

Comment:
This is what i wanted to make sure is working for the worked to ask for a source